roles/dns_adblocking/tasks/main.yml

```diff
@@ -8,7 +8,7 @@
     - name: Dnsmasq installed
       package: name=dnsmasq
 
-    - name: Ensure that the dnsmasq user exist
+    - name: Ensure that the dnsmasq user exists
```

Comment:
This task should be deprecated and deleted

Reply:
Deleted it